lib/upipe-ts/upipe_rtp_fec.c

```diff
@@ -551,8 +551,7 @@ static void clear_fec(struct upipe *upipe)
     upipe_rtp_fec->last_seqnum = UINT32_MAX;
     upipe_rtp_fec->latency = 0;
 
-    memset(upipe_rtp_fec->recent, 0xff, 2 * upipe_rtp_fec->rows *
-            upipe_rtp_fec->cols * sizeof(*upipe_rtp_fec->recent));
+    memset(upipe_rtp_fec->recent, 0xff, sizeof(upipe_rtp_fec->recent));
 }
```

Comment:
This is not needed, all accesses to ->recent are done modulo 2 * rows * col already